test/e2e/test-request-skipNav/test.mjs

```diff
@@ -0,0 +1,56 @@
+import { Actor } from 'apify';
+import { purgeLocalStorage } from '@crawlee/utils';
+import { CheerioCrawler, log, Request } from '@crawlee/cheerio';
+import { initialize, expect } from '../tools.mjs';
+
+await initialize(import.meta.url);
+
+let requestCounter = 0;
+let navigationCounter = 0;
+
+// Persisting internal settings of `Request`.
+await Actor.main(async () => {
+    await purgeLocalStorage();
```

Comment:
Purge should be the default in v3, yeah. I would add it to `init` as well, since `main` should be just syntax sugar for `init` and `exit` so the behavior should be exactly the same